tests/test_generate_database_data.py

```diff
@@ -70,40 +71,53 @@ def test_set_all_frameworks_to_expired(self):
         self.api_client.update_framework.assert_any_call("g-cloud-7", {"status": "expired"})
 
 
-class TestCreateGCloudFramework(TestGenerateDataBase):
+class TestOpenGcloud12(TestGenerateDataBase):
 
     def test_passed_valid_data(self):
-        add_live_g_cloud_framework(self.api_client)
+        open_gcloud_12(self.api_client)
         self.api_client.create_framework.assert_called_with(
             slug=G_CLOUD_FRAMEWORK["slug"],
             name=G_CLOUD_FRAMEWORK["name"],
             framework_family_slug=G_CLOUD_FRAMEWORK["family"],
             lots=[lot["slug"] for lot in G_CLOUD_FRAMEWORK["lots"]],
             has_further_competition=False,
             has_direct_award=True,
-            status="live"
+            status="open"
         )
-
         self.api_client.update_framework.assert_called_with(
             framework_slug=G_CLOUD_FRAMEWORK["slug"],
             data={
-                "allowDeclarationReuse": G_CLOUD_FRAMEWORK["allowDeclarationReuse"],
-                "applicationsCloseAtUTC": G_CLOUD_FRAMEWORK["applicationsCloseAtUTC"],
-                "intentionToAwardAtUTC": G_CLOUD_FRAMEWORK["intentionToAwardAtUTC"],
-                "clarificationsCloseAtUTC": G_CLOUD_FRAMEWORK["clarificationsCloseAtUTC"],
-                "clarificationsPublishAtUTC": G_CLOUD_FRAMEWORK["clarificationsPublishAtUTC"],
-                "frameworkLiveAtUTC": G_CLOUD_FRAMEWORK["frameworkLiveAtUTC"],
-                "frameworkExpiresAtUTC": G_CLOUD_FRAMEWORK["frameworkExpiresAtUTC"]
+                "clarificationQuestionsOpen": True
             }
         )
 
-    def test_doesnt_add_if_already_created(self):
+    def test_doesnt_add_gcloud_if_already_present(self):
         self.api_client.find_frameworks.return_value = {
             "frameworks": [
                 {
                     "slug": G_CLOUD_FRAMEWORK["slug"]
                 }
             ]
         }
-        add_live_g_cloud_framework(self.api_client)
+        open_gcloud_12(self.api_client)
         assert not self.api_client.create_framework.called
+
+        self.api_client.update_framework.assert_called_with(
+            framework_slug=G_CLOUD_FRAMEWORK["slug"],
+            data={
+                "clarificationQuestionsOpen": True
+            }
+        )
+
+
+class TestMakeGcloud12Live(TestGenerateDataBase):
```

Comment:
```suggestion
class TestMakeGCloud12Live(TestGenerateDataBase):
```

Again, I think this is the preferred way to camel-case G-Cloud